master/pkg/schemas/unions.go

```diff
@@ -0,0 +1,73 @@
+package schemas
+
+import (
+	"reflect"
+)
+
+func derefInput(val reflect.Value) (reflect.Value, bool) {
```

Comment:
I'm also ok just deleting this file and inlining the code where it is called.  Since I wrote it, I've concluded a much better strategy is to never have common members on union types, which is the only purpose of this file anyway.